R/add_section.R

```diff
@@ -11,11 +11,15 @@
 #' or within a section as a child document.
 #' @export
 #'
-#' @examples add_section(
-#'   new_section = "Ecosystem Considerations", section_location = "after-discussion",
+#' @examples
+#'\dontrun{
+#' add_section(
+#'   new_section = "Ecosystem Considerations",
+#'   section_location = "after-discussion",
 #'   custom_sections = c("introduction.qmd", "model.qmd", "results.qmd", "discussion.qmd"),
```

Comment:
This might need to be a list. Can you check this? @sbreitbart-NOAA

Reply:
Running the examples with `custom_sections = c("introduction.qmd"...),` and `custom_sections = list("introduction.qmd"...` both seem to provide the same output in the SAR_species_skeleton.qmd. Would you prefer I use one option over the other? Or even use one format in one example, and the other format in another example?